force_wfmanager/cli/tests/test_execution.py

```diff
@@ -0,0 +1,31 @@
+import unittest
+import subprocess
+
+
+class TestExecution(unittest.TestCase):
+
+    def test_command_line_invocation(self):
+        """Test if the workflow manager can be loaded from the command line"""
+
+        # Does the command exist?
+        available_commands = subprocess.check_output(['/bin/bash', '-c', 'compgen -c'])
```

Comment:
this is not portable, but in any case you don't need it. The test will fail anyway if the command is not available.
Just invoke it